File: JavaProjects/ClasePOO_6256_2/src/Mamifero.java
public class Mamifero {

    // Propiedades
    private String nombre;

    // Constructor
    public Mamifero(String nombre) {
        this.nombre = nombre;
    }
    public Mamifero () {
        this.nombre = "Sin definir";
    }

    // Métodos
    public String expresarse(){
        return "Por definir";
    }

    //Getter y Setter
    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}

File: C++Projects/6256_cppclase1-2_archivosdistintos/Mamifero.cpp
#include "Mamifero.h"

// Definir el constructor de la clase
Mamifero::Mamifero(string nombre){
    this->nombre = nombre;
    cout << "Ha creado un mamífero llamado " << nombre << endl;
}

// Definir el método propio de la clase
string Mamifero::amamantar(){
    return "Consumo leche proveniente de mi madre";
}

// Definir los Getter y Setter
string Mamifero::getNombre(){
    return nombre;
}
void Mamifero::setNombre(string nombre){
    this->nombre = nombre;
}